setup/ubuntu14.04/ansible-playbook.yaml

```diff
@@ -28,6 +27,7 @@
     - name: User | Download pubkey(s)
       get_url: url=https://github.com/{{ item }}.keys dest=/tmp/{{ item }}.keys
       delegate_to: 127.0.0.1
+      become: no
```

Comment:
I've just ignored these parts of the playbook since we nowadays use the shared pubkeys (Ignored while running the playbook by stepping it through). I don't have a problem with this change but it's probably pointless.